src/libraries/System.Private.CoreLib/src/System/Decimal.DecCalc.cs

```diff
@@ -184,20 +184,7 @@ private static ulong UInt32x32To64(uint a, uint b)
 
             private static void UInt64x64To128(ulong a, ulong b, ref DecCalc result)
             {
-                ulong low = UInt32x32To64((uint)a, (uint)b); // lo partial prod
```

Comment:
Can you delete `UInt32x32To64` as well?

Reply:
Could yes, but:
- Unfortunately does not exist any BigMul method for two uint32.
- It is used in a lot of places where narrowing/casting from ulongs to uints are done, and the code is pretty convoluted already. This shows intent quite nicely

Reply:
Did you measure the performance of this change? Last time I checked, a similar change caused a significant performance regression because the codegen for `Bmi2.MultiplyNoFlags` is suboptimal.